### DISCUSSION TITLE
Doors/beds glitch out with no reason

### DISCUSSION THREAD

**Discussion by Stellrow** [Pryaxis/TShock#1841] [General]
There is sometime's a chance when a player walks in a house(and the auto-open-close) happens the door will glitch out and no longer existing for that player

Also for some reason placing a bed next to a door,once the door is opened the bed glitches out,the player gets the bed as if it was mined but the bed is still there

I switched to the vanilla server and it has no problems

**Comment by hakusaro** [comment]
This is a known issue with smart doors. We're tracking it in #1774 and in other places. It should be fixed eventually!
